web-client/src/app/utils/camara.helpers.ts

```diff
@@ -0,0 +1,12 @@
+import { Capacitor } from '@capacitor/core';
+
+export const deviceHasCamera = async () => {
+  if (Capacitor.isNativePlatform()) {
+    return Capacitor.isPluginAvailable('Camera');
+  } else {
+    const devices = await navigator?.mediaDevices.enumerateDevices();
+    console.log(devices);
```

Comment:
Is this `console.log()` part of debugging?
If so, perhaps it should be removed?